unlock-app/src/components/interface/checkout/main/Minting.tsx

```diff
@@ -148,6 +172,69 @@ export function Minting({
               <Icon icon={ExternalLinkIcon} size="small" />
             </a>
           )}
+          {tokenId && isEthPassSupported(lock!.network) && (
+            <ul className="grid grid-cols-2 gap-3 pt-4">
+              {!isIOS && (
+                <li className="">
+                  <AddToDeviceWallet
+                    className="w-full px-2 h-8 text-xs grid grid-cols-[20px_1fr] rounded-md bg-black text-white"
+                    iconLeft={
+                      <Image
```

Comment:
Does Image Component work outside of vercel? Can you re-check on yarn build and yarn start?